### ISSUE TITLE
Setting initial solution in highspy

### ISSUE THREAD

**Issue by jajhall** [ERGO-Code/HiGHS#1936]
### Discussed in https://github.com/ERGO-Code/HiGHS/discussions/1933

<div type='discussions-op-text'>

<sup>Originally posted by **Dimjoy** September 18, 2024</sup>
Hello!
As far as I understand, currently there are two call signatures for `setSolution` method (according to _core.pyi file):
```
@overload
def setSolution(self, arg0: HighsSolution) -> HighsStatus: ...
@overload
def setSolution(self, arg0: int, arg1: numpy.ndarray[numpy.int32], arg2: numpy.ndarray[numpy.float64]) -> HighsStatus: ...
```
Apparently, the second mimics `Highs_setSparseSolution` method from the C API. I am trying to use it for a simple LP-problem from the Examples section of the Guide (2 continuous variables and 2 constraints) with the following line of code:
```
h.setSolution(1, np.array([0]), np.array([3.1]))
```
which fails with the following error message:
>TypeError: setSolution(): incompatible function arguments. The following argument types are supported:
    1. (self: highspy._core._Highs, arg0: HighsSolution) -> highspy._core.HighsStatus
Invoked with: <highspy.highs.Highs object at 0x00000229BCD88AC0>, 1, array([0]), array([3.1])

Am I doing it wrong? Or setting partial solution is not supported right now in the Python API?</div>

**Comment by mathgeekcoder** [comment]
FYI: I can reproduce this issue with the currently released version of highspy.  However, it seems to work with the latest changes in #1942.

**Comment by jajhall** [comment]
Closed by #1942
